### PR TITLE
GH-633 - Updating active indicator for boards

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
GH-633 - Updating active indicator for boards

#### Ticket Link
https://github.com/mattermost/focalboard/issues/633

#### Screenshot
Screenshot 2021-07-22 at 5 03 09 PM
Screenshot 2021-07-22 at 5 04 34 PM

### CODE REVIEW

**Review by harshilsharma63 — APPROVED** [review]
LGTM!

File: webapp/src/components/sidebar/sidebarBoardItem.tsx [hunk 1]
```diff
@@ -92,10 +93,14 @@ const SidebarBoardItem = React.memo((props: Props) => {
     const displayTitle: string = board.title || intl.formatMessage({id: 'Sidebar.untitled-board', defaultMessage: '(Untitled Board)'})
     const boardViews = sortBoardViewsAlphabetically(views.filter((view) => view.parentId === board.id))
 
+    console.log('####################################################################')
+    console.log(props.activeViewId)
+    console.log('####################################################################')
+
```
Comment (hahmadia) [inline webapp/src/components/sidebar/sidebarBoardItem.tsx:99]: I think these should be removed?

**Review by hahmadia — APPROVED** [review]
LGTM

File: webapp/src/components/sidebar/sidebarBoardItem.tsx (hunk 1 above)
Comment (asaadmahmood) [reply webapp/src/components/sidebar/sidebarBoardItem.tsx:99]: @hahmadia Harshil just added those for me to add the active indicator on the view as well.

**Pull request merged**